install.ps1

```diff
@@ -845,18 +846,25 @@ refreshenv
 Write-Host "[+] Installing shared module..."
 choco install common.vm -y --force
 refreshenv
+
+# Use single config
```

Comment:
At least at this point, I think it is the easiest alternative (also for user modifying the config) to have a single config file. We can discuss this further after this has been merged (maybe also with commando-vm)